browser/index.js

```diff
@@ -839,7 +842,12 @@ module.exports = module.exports = {
                         categories={categoriesOverall ? categoriesOverall : []}
                         onApply={_self.onApplyLayersAndChemical}
                         onClose={onCloseHandler}
-                    /></Provider>, document.getElementById(introlModalPlaceholderId));
+                    /></Provider>, document.getElementById(introlModalPlaceholderId)); */
+                ReactDOM.render(<ThemeProvider>
+                    <Dialogue titleText='Welcome to Calypso'
+                        onCloseButtonClick={onCloseHandler}>
+                        <div>This is test for modal content</div>
```

Comment:
Remove child content from feature-component.
Child content should be defined inside the component.